### PR TITLE
convert SeaBreeze to twig via `zikula-tools toTwig:convert`

### PR DESCRIPTION
**Pull request by craigh**
| Q                 | A
| ----------------- | ---
| Bug fix?          | no
| New feature?      | no
| BC breaks?        | no
| Deprecations?     | no
| Tests pass?       | yes
| Fixed tickets     | -
| Refs tickets      | #1753
| License           | MIT
| Doc PR            | -
| Changelog updated | yes

### CODE REVIEW

**Comment by craigh** [comment]
@Guite @cmfcmf I [fixed](https://github.com/zikula/Tools/commit/ddc7c6584a7b43f24199719b075a0bb25006ef35) the modurl converter and re-ran the script to produce this PR instead.

File: src/themes/SeaBreezeTheme/Resources/views/includes/adminheader.html.twig [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<div id="topline" class="clearfix">
+    <div class="pull-left">{userwelcome|ucwords}</div>
+    <div class="pull-right">{{ showblockposition('search') }}</div>
+</div>
+<div id="header" class="clearfix">
+    <h1>{# adminpanelmenu #} <a href="{{ app.request.baseUrl }}" title="{{ {{ getModVar('ZConfig', 'slogan') }} }}">{{ {{ getModVar('ZConfig', 'sitename') }} }}</a></h1>
```
Comment (cmfcmf) [inline src/themes/SeaBreezeTheme/Resources/views/includes/adminheader.html.twig]: The modvar thing looks wrong.

**Comment by craigh** [comment]
@cmfcmf I [fixed](https://github.com/zikula/Tools/commit/e0f2f8bfe4cba994078b2278e95cf54b0e3a331f) the 'modvar thing' and reset/pushed a new version to this branch. please re-inspect.

File: src/themes/SeaBreezeTheme/Resources/views/blocks/lsblock.html.twig [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<!-- Start Left Block Start -->
+{% if not title is empty  %}
```
Comment (craigh) [inline src/themes/SeaBreezeTheme/Resources/views/blocks/lsblock.html.twig]: this is wrong

**Comment by craigh** [comment]
fixed the `is not empty` problem

**Comment by cmfcmf** [comment]
As you see [here](https://github.com/zikula/core/pull/2592/files?diff=split#diff-c2be4ba6f400cb40a9ce1b02ac3bed2aR1), git doesn't recognize that the files have been renamed but rather thinks you removed all the *.tpl files and added all the *.html.twig files. This makes it difficult to compare the changes. I don't know if there is something one could do to avoid it, because it seemed to work [here](https://github.com/zikula/core/pull/2592/files?diff=split#diff-946b4753e5199d93647abddefb4a21b6L1).

**Comment by craigh** [comment]
yes. I am aware of that. I'm not sure why it treats some differently than others or what could be done.

File: src/themes/SeaBreezeTheme/Resources/views/Include/userheader.html.twig [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<div id="topline" class="clearfix">
+    <div class="pull-left">{# userwelcome|ucwords #}</div>
+    <div class="pull-right">{{ showblockposition('search') }}</div>
+</div>
+<div id="header" class="clearfix">
+    <h1>{# adminpanelmenu #} <a href="{{ app.request.baseUrl }}" title="{{ getModVar('ZConfig', 'slogan') }}">{{ getModVar('ZConfig', 'sitename') }}</a></h1>
```
Comment (cmfcmf) [inline src/themes/SeaBreezeTheme/Resources/views/Include/userheader.html.twig]: Do we need a replacement for adminpanelmenu?

File: src/themes/SeaBreezeTheme/Resources/views/modules/ZikulaBlocksModule/Block/Extmenu/topnav.html.twig [hunk 4]
```diff
@@ -0,0 +1,12 @@
+{menu from=$menuitems item='item' name='extmenu' id='nav'}
```
Comment (cmfcmf) [inline src/themes/SeaBreezeTheme/Resources/views/modules/ZikulaBlocksModule/Block/Extmenu/topnav.html.twig]: Do we need a replacement?

File: src/themes/SeaBreezeTheme/Resources/views/modules/ZikulaSearchModule/Block/search.html.twig [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+        <input id="block_search_q" type="search" name="q" size="20" maxlength="255" results="10" autosave="Search" class="theme_search_input" />
+        {% if vars.displaySearchBtn == 1 %}
+        <input class="theme_search_button" type="submit" value="{gt text="Search" domain='zikula'}" />
+        {% endif %}
+        <div style="display: none;">
+            {% for plugin, plugin_option in plugin_options %}
+            {{ plugin_option }}
+            {% endfor %}
+        </div>
+        {searchvartofieldnames data=$modvars.ZikulaSearchModule prefix="modvar" assign="modvariables"}
```
Comment (cmfcmf) [inline src/themes/SeaBreezeTheme/Resources/views/modules/ZikulaSearchModule/Block/search.html.twig]: Same here.

File: src/themes/SeaBreezeTheme/Resources/views/Include/userheader.html.twig (hunk 3 above)
Comment (craigh) [inline src/themes/SeaBreezeTheme/Resources/views/Include/userheader.html.twig]: I will be making one, yes. just not done yet.

File: src/themes/SeaBreezeTheme/Resources/views/modules/ZikulaSearchModule/Block/search.html.twig [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+        {% for name, value in modvariables %}
+        <input type="hidden" name="{$name|safetext}" value="{$value|safetext}" />
```
Comment (cmfcmf) [inline src/themes/SeaBreezeTheme/Resources/views/modules/ZikulaSearchModule/Block/search.html.twig]: and here

File: src/themes/SeaBreezeTheme/Resources/views/modules/ZikulaSearchModule/Block/search.html.twig [hunk 7]
```diff
@@ -0,0 +1,18 @@
+<form id="theme_search" method="post" action="{{ path('search_user_search') }}{# @todo probably an incorrect path #}">
+    <div>
+        <input id="block_search_q" type="search" name="q" size="20" maxlength="255" results="10" autosave="Search" class="theme_search_input" />
+        {% if vars.displaySearchBtn == 1 %}
+        <input class="theme_search_button" type="submit" value="{gt text="Search" domain='zikula'}" />
```
Comment (cmfcmf) [inline src/themes/SeaBreezeTheme/Resources/views/modules/ZikulaSearchModule/Block/search.html.twig]: This should have been converted automatically, right?

Comment (craigh) [inline src/themes/SeaBreezeTheme/Resources/views/modules/ZikulaSearchModule/Block/search.html.twig]: no, the domain parameter failed the auto-conversion

File: src/themes/SeaBreezeTheme/Resources/views/modules/ZikulaSearchModule/Block/search.html.twig (hunk 6 above)
Comment (craigh) [inline src/themes/SeaBreezeTheme/Resources/views/modules/ZikulaSearchModule/Block/search.html.twig]: I really appreciate you finding all my mistakes :boat:

Comment (craigh) [inline src/themes/SeaBreezeTheme/Resources/views/modules/ZikulaSearchModule/Block/search.html.twig]: all three of the items you just referenced are in template overrides and will not be used until the module itself is updated to use a twig template. so, I feel we can merge this.

File: src/themes/SeaBreezeTheme/Resources/views/modules/ZikulaSearchModule/Block/search.html.twig (hunk 7 above)
Comment (craigh) [inline src/themes/SeaBreezeTheme/Resources/views/modules/ZikulaSearchModule/Block/search.html.twig]: this was [addressed here](https://github.com/zikula/Tools/commit/5a7a78d97ab8b78a57829afa5068211c3d237717)

Comment (craigh) [inline src/themes/SeaBreezeTheme/Resources/views/modules/ZikulaSearchModule/Block/search.html.twig]: and [here](https://github.com/zikula/core/commit/eb4112e0d8741a73e3a2d10118c24698e6c40925)

**Pull request merged**